src/widgets-service.js

```diff
@@ -185,7 +194,8 @@ WidgetsService.prototype.createTimeSeriesModel = function (attrs, layer) {
 
   var widgetModel = new WidgetModel({
     id: attrs.id,
-    type: 'time-series'
+    type: 'time-series',
+    dataviewModelAttrsNames: ['column', 'columnType', 'bins', 'start', 'end']
```

Comment:
It's due to the [cartodb.js](https://github.com/CartoDB/cartodb.js/blob/v4/src/dataviews/histogram-dataview-model.js#L10) code, I'll send a PR there to update the camelCased ones to have this fixed then so we're consistent.